### ISSUE TITLE
I am trying to find out when Redux Toolkit was released

### ISSUE THREAD

**Issue by nonopolarity** [reduxjs/redux-toolkit#888]
I am trying to find out when Redux Toolkit was released but can't find the info.  The 1.3.0 release was beyond Beta about 2020 March, and people did blog about it in 2020 January.  What about 1.2 or 1.1, were those all Alphas and Betas?

When is it officially beyond Beta?

**Comment by phryneas** [comment]
October 2019, when it hit 1.0 - back then it was named "redux-starter-kit" and was renamed shortly thereafter.

https://blog.isquaredsoftware.com/2019/10/redux-starter-kit-1.0/

**Comment by nonopolarity** [comment]
and so is it true we use forget those mapDispatchToProps() and connect(...)(), and ... not only use the useDispatch() and useSelector(), but outright totally follow Redux Toolkit as the common practice methodology?

**Comment by phryneas** [comment]
You might be interested in the official Redux Style guide.

https://redux.js.org/style-guide/style-guide/


Generally yes, we recommend the react-redux hooks over connect, as it is easier to learn for people familiar with hooks (which should by now be every React developer, at least to some degree) and easier to use with TypeScript. Connect is not going away anytime soon, but it is more complicated to use.

And yes, Redux Toolkit is the officially recommended approach. See the style guide linked above.

**Comment by nonopolarity** [comment]
aha, thank you. I see this part:

> Use Redux Toolkit for Writing Redux Logic#
Redux Toolkit is our recommended toolset for using Redux. It has functions that build in our suggested best practices, including setting up the store to catch mutations and enable the Redux DevTools Extension, simplifying immutable update logic with Immer, and more.

**Comment by markerikson** [comment]
What specific questions are you trying to answer here?  What's the background on this request?

**Comment by nonopolarity** [comment]
> What specific questions are you trying to answer here? What's the background on this request?

Actually, I was thinking can the docs of Redux Toolkit add some introduction as to when it was. released, a little background, and who is recommending the use of Redux Toolkit over other style? It is the Redux team itself recommending it?

**Comment by markerikson** [comment]
I mean yes, we wrote it, it's an official package, and we specifically say in our docs you should use it :)

**Comment by nonopolarity** [comment]
> I mean yes, we wrote it, it's an official package, and we specifically say in our docs you should use it :)

could there be an intro, like historic background, release dates. I tried to find a Wikipedia page for it and didn't.

**Comment by markerikson** [comment]
Yeah, probably the best option for now would be to link my "RTK 1.0" blog post from the introduction page.

**Comment by akash-shrestha** [comment]
same here
